pkg/lib/cockpit.d.ts

```diff
@@ -247,7 +247,7 @@ declare module 'cockpit' {
     }
 
     interface FileHandle<T> {
-        read(): Promise<T>;
+        read(callback: (data: T | null) => T, tag?: FileTag): Promise<T>;
```

Comment:
`read` doesn't have a callback parameter. Or am I misreading the type annotations?

I wouldn't be surprised if we can't type `read` correctly. It would be something like `read(): Promise<T, FileTag>`, no?

Reply:
Oh, `modify` has been annotated as `: Promise<[T, FileTag]>`. Does that work?